nbconvert/tests/files/notebook2.ipynb

```diff
@@ -1569,6 +1569,8 @@
    "metadata": {},
    "source": [
     "```python\n",
+    "import math, numpy\n",
+    "def matrix_calc(L):\n",
```

Comment:
Why is there also a function definition? There's a function defined immediately below this line, so technically this is a syntax error... unless that's what you want to test for?